roles/zabbix_web/handlers/main.yml

```diff
@@ -8,6 +8,8 @@
   become: true
   when:
     - zabbix_websrv == 'apache'
+    - ansible_facts.services[_apache_service] is defined
+    - ansible_facts.services[_apache_service].state == "running"
```

Comment:
It looks like we use this handler as both restarting and starting the service initially.

Since the first install doesn't start the service for us it is not "running" and this handler gets skipped. I believe that is why your tests are failing

Reply:
I removed it from the web files. I have issues running molecule locally and the original issue was for the iptables/firewall, so not really needed now. Thank you for you comment! 👍